applications/display/display-plugins/org.csstudio.display.pvtable/.classpath

```diff
@@ -1,7 +1,18 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<classpath>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
-	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
-	<classpathentry kind="src" path="src/"/>
-	<classpathentry kind="output" path="target/classes"/>
-</classpath>
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry exported="true" kind="lib" path="libs/xmlbeans-2.6.0.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/lib/commons-codec-1.9.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/lib/commons-logging-1.1.3.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/lib/junit-4.12.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/lib/log4j-1.2.17.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/poi-3.13-20150929.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/poi-examples-3.13-20150929.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/poi-excelant-3.13-20150929.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/poi-ooxml-3.13-20150929.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/poi-ooxml-schemas-3.13-20150929.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/poi-scratchpad-3.13-20150929.jar"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
```

Comment:
are any of these in orbit?  If not, they should be added to the maven-osgi-bundles repo (which wraps artifacts from maven central into bundles and puts them into our p2).